polkadot/runtime/rococo/constants/src/lib.rs

```diff
@@ -100,6 +100,26 @@ pub mod fee {
 	}
 }
 
+/// System Parachains.
+pub mod system_parachain {
+	use xcm::latest::prelude::*;
+
+	/// Network's Asset Hub parachain ID.
+	pub const ASSET_HUB_ID: u32 = 1000;
+	/// Contracts parachain ID.
+	pub const CONTRACTS_ID: u32 = 1002;
+	/// Encointer parachain ID.
+	pub const ENCOINTER_ID: u32 = 1003;
+	/// BridgeHub parachain ID.
+	pub const BRIDGE_HUB_ID: u32 = 1013;
+
+	frame_support::match_types! {
+		pub type SystemParachains: impl Contains<MultiLocation> = {
+			MultiLocation { parents: 0, interior: X1(Parachain(ASSET_HUB_ID | CONTRACTS_ID | ENCOINTER_ID | BRIDGE_HUB_ID)) }
```

Comment:
When this is imported by `asset-hub-rococo`, this `parents: 0` will be wrong from the point of view of the parachain.

Reply:
I created a new one instead of importing this one